tools-local/unix/cmake/get-version.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+usage()
+{
+    echo "usage: $0 <repository-root> cmake <override-scripts-folder-path> <strict-tool-version-match> <tool-path>"
+    echo "  repository-root: This argument is ignored."
+    echo "  override-scripts-folder-path: This argument is ignored."
+    echo "  strict-tool-version-match: This argument is ignored."
+    echo "  tool-path: Path to CMake executable or the folder containing the executable."
+    echo "If successful then, returns the version number of CMake executable."
+    echo "Exit 1 if the executable is not available at the specified path or folder."
+}
+
+if [ $# -ne 5 ]; then
+    usage
+    exit 1
+fi
+
+if [ "$2" != "cmake" ]; then
+    echo "Second argument should be cmake."
+    usage
+    exit 1
+fi
+
+if [ ! -z "$5" ] && [ ! -f "$5" ]; then
```

Comment:
Yes, empty string check is not required.

---
In reply to: [110415788](https://github.com/ravimeda/corefx/pull/9#discussion_r110415788) [](ancestors = 110415788)